src/cdk/table/table.spec.ts

```diff
@@ -874,6 +875,22 @@ describe('CdkTable', () => {
       });
     }
 
+    function expectStickyBorderClass(element: any, directions: {[key: string]: boolean} = {}) {
+      ['top', 'bottom', 'left', 'right'].forEach(d => {
+        const stickyClass = `cdk-table-sticky-border-elem-${d}`;
+
+        if (directions[d]) {
+          // If no expected position for this direction, must either be unset or empty string
+          expect(element.classList.contains(stickyClass)).toBe(true,
```

Comment:
Instead of asserting as `expect(elem.classList.contains(foo)).toBe(true)`, you can use `expect(elem.classList).toContain(foo)` which produces a more readable failure message.